extensions/big-data-cluster/src/bigDataCluster/utils.ts

```diff
@@ -273,3 +271,34 @@ export function getControllerEndpoint(serverInfo: azdata.ServerInfo): string | u
 export function getBdcStatusErrorMessage(error: Error): string {
 	return localize('endpointsError', "Unexpected error retrieving BDC Endpoints: {0}", error.message);
 }
+
+export const bdcConfigSectionName = 'bigDataCluster';
+export const ignoreSslConfigName = 'ignoreSslVerification';
+
+/**
+ * Retrieves the current setting for whether to ignore SSL verification errors
+ */
+export function getIgnoreSslVerificationConfigSetting(): boolean {
+	return _ignoreSslVerification;
+}
+
+let _ignoreSslVerification = false;
```

Comment:
It's really cheap to query so generally we don't bother caching this kind of thing globally.

One nitpick with this design: it makes things much harder to unit tests (e.g. with/without the setting) in the future. Not a deal breaker, but would be nice to have this be more modular

Reply:
Yeah, maybe I'm overoptimizing a bit here. I'll change it to just query directly each time.

Reply:
VScode's configuration apis keep the exthost side of things updated with the mainthread side, so you won't be making rpc calls when doing config lookups.

Reply:
Good to know, thanks @anthonydresser - that was definitely one of the concerns around perf.